CorsixTH/Lua/languages/brazilian_portuguese.lua

```diff
@@ -58,6 +58,36 @@ vip_names = {
  [6] = "O Presidente da Cruz Vermelha",
 }
 
+-- An override for the squits becoming the the squits see issue 1646
+adviser.research.drug_improved_1 = "Seu Centro de Pesquisa melhorou o medicamento para %s"
+
+-- Disease overrides where there are typos
+golf_stones.cure = "Cura - Dois cirurgiöes operam para extrair os cálculos."
+ruptured_nodules.cure = "Cura - Dois cirurgiöes qualificados colocam certas partes com mäos firmes."
+slack_tongue.cause = "Causa - Fazer muita fofoca."
+slack_tongue.cure = "Cura - Coloca-se a língua no corta-línguas e elimina-se de forma rápida, eficaz e dolorosa."
+the_squits.cure = "Cura - Uma mistura pegajosa de substâncias viscosas preparada na farmácia solidifica as tripas do paciente."
+bloaty_head.cure = "Cura - Fura-se a cabeça inflada, em seguida, volta-se a inflar até o tamanho correto com uma máquina inteligente."
+
+-- Rooms overrides where there are typos
+inflation[2] = "Os pacientes que sofrem a dolorosa doença de terem a cabeça inflada devem ir à sala de inflatoterapia, será desinflada sua cabeça e em seguida vota a ser inflada à pressäo correta.//"
+staff_room[2] = "Os seus funcionários se cansam quando realizam o seu trabalho. Precisam desta sala para descansar e se refrescar. Os funcionários cansados säo mais lentos, pedem mais dinheiro e por fim väo embora. Também cometem mais erros. Vale a pena construir uma sala que tenham muitos passatempos. Assegure-se de que há lugar para vários funcionários ao mesmo tempo. "
+
+-- Staff description overrides where there are typos
+staff_descriptions.bad[14] = "Pessoa ardilosa, matreira e subversiva. "
+staff_descriptions.misc[11] = "Destila uísque. "
+
+-- Correction to a pay rise prompt with typos
+pay_rise.regular[1] = "Estou esgotado. Preciso de umas boas férias, mais um aumento de %d, se näo quiser deixo este maldito emprego."
+
+-- Level description overrides where there are typos. Note: This is the only portion of the game that SHOULD use double space after fullstops etc.
```

Comment:
Introduction texts must have double spaces after the full-stop.

Reply:
Not all languages have the same rules. This is consistent as far as I see.

Reply:
Should the comment on line 83 be partly removed then for brazilian_portuguese?